content/docs/1.5/scalers/cron.md

```diff
@@ -27,6 +27,16 @@ triggers:
 - `timezone` is one of the acceptable values from the IANA Time Zone Database. The list of timezones can be found in: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones. Required.
 - `start` is the cron expression indicating the start of the cron schedule. Required.
 - `end` is the cron expression indicating the end of the cron schedule. Required.
+
+> **Notice:**
+> **Start and end should not be same.**
+>
+> For example, the following schedule is not valid:
+> ```yaml
+> start: 30 * * * *
+> end: 30 * * * *
+>```
+
 - `desiredReplicas` is the number of replicas to which the resource has to be scaled between the start and end of the cron schedule. Required.
```

Comment:
I've missed this in the initial review but let's list all parameters first and then drop the notice.

```suggestion
- `end` is the cron expression indicating the end of the cron schedule. Required.
- `desiredReplicas` is the number of replicas to which the resource has to be scaled between the start and end of the cron schedule. Required.

> **☝ Notice - Start and end should not be same.**
>
> For example, the following schedule is not valid:
> ```yaml
> start: 30 * * * *
> end: 30 * * * *
>```

```

Reply:
@tomkerkhove  sure thing.